src/components/Preview/Modal.css

```diff
@@ -1,8 +1,7 @@
 .preview--screen {
   width: 100vw;
   height: 100vh;
-  /* transition animation when opened */
-  transform: translate(100%, 0);
+  display: none;
```

Comment:
Why was it translated like that? This was making the progress bar appear on the right side of the container.